README.md

```diff
@@ -21,74 +23,135 @@ To learn more, see the [DC/OS Overview](https://dcos.io/docs/latest/overview/).
 - Report an Issue - <https://dcosjira.atlassian.net>
 - Contribute - <https://dcos.io/contribute/>
 
-# What's In This Repo?
 
-DC/OS itself is composed of many individual components precisely configured to work together in concert.
+# Releases
 
-This repo contains the release and package building tools necessary to produce installers for various on-premises and cloud platforms.
+DC/OS releases are publicly available on <http://dcos.io/releases/>
 
-| Directory | Contents |
-| --------- | -------- |
-| *docker*             | Locally defined docker containers packages are built in
-| *docs*               | Documentation
-| *dcos_installer*     | Backend for Web, SSH, and some bits of the Advanced installer. Code is being cleaned up
-| *gen*                | Python library for rendering yaml config files for various platforms into packages, with utilities to do things like make "late binding" config set by CloudFormation
-| *gen/build_deploy*   | Code to take a build and transform it into a particular platform deployment tool (Bash / command line, AWS, Azure, etc.)
-| *packages*           | Packages which make up DC/OS (Mesos, Marathon, AdminRouter, etc). These packages are built by pkgpanda, and combined into a "bootstrap" tarball for deployment.
-| *pkgpanda*           | DC/OS baseline/host package management system. Tools for building, deploying, upgrading, and bundling packages together which live on the root filesystem of a machine / underneath Mesos.
-| *pytest*             | Misc. tests. Should be moved to live next to the appropriate code
-| *release*            | Release tools for DC/OS. (Building releases, building installers for releases, promoting between channels)
-| *ssh*                | AsyncIO based parallel ssh library used by the installer
-| *test_util*          | various scripts, utilities to help with integration testing
+Release artifacts are managed by Mesosphere on Amazon S3, using a CloudFront cache, under the path <https://downloads.dcos.io/dcos/>
```

Comment:
I get AccessDenied when I try to access this URL. Is it worth linking this when we have stable/EA/master links below?